lib/surface_site_web/live/events.ex

```diff
@@ -92,29 +95,174 @@ defmodule SurfaceSiteWeb.Events do
-                  </#Raw>
+                  <#Code
+                    language="elixir"
+                    module={{ SurfaceSiteWeb.Events.SurfaceCounter }}
+                    show_line_numbers
+                    line_range={{ 2..23 }}
+                    selected_lines="13-15"
+                  />
                 </footer>
               </div>
 
               <#Markdown>
                 As you can see, we didn't have to define `phx-target` for any of the buttons. Sweet!
 
+                > **Note**: The complete list of available events, as well as other types of bindings, can be found at
+                section [Bindings](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html#module-bindings)
+                in the Phoenix LiveView's docs.
+
+              </#Markdown>
+
+              <#Markdown>
+
+                ## Pass event through an `event` property
+
                 Another great thing about Surface's approach is that it makes passing events as properties
                 also more intuitive. Using phoenix templates, unless you always pass both, the event and the target,
                 you cannot be sure where the event will be handled. You need to know upfront if there's a `phx-target`
-                defined for that DOM element inside that component.
+                defined for that DOM element inside that component. Using Surface, the event is **always** passed
+                along with the related target, assuming, by default, that the target is the caller component/view.
+
+                In the above examples the events have been handled by the component itself.
+                Sometime the component does not know how to handle an event and you would like to handle this event in the parent component.
+                For that kind of use case, you must declare the event in the child component by using the `prop` macro defining the type as `:event`,
+                and pass the value of that prop to the `:on-[event]` directive.
+              </#Markdown>
+
+                See the properties <LiveRedirect label="Event" to="/properties#event-property" /> section for more details about event properties.
+
+              <#Markdown>
+
+                ### Stateless component
+                The simplest case you need to pass an event is when you create a stateless component that includes an element
+                that defines a server binding (event). Since the component is stateless, it cannot handle the event by itself
+                so it needs to receive the event handler as a property.
+
+                For example, imagine a `Button` stateless component that raises an event when the user clicks on it.
```

Comment:
What's the most precise verb to use here for dispatching/emitting/triggering an event? I think there might be something more fitting than `raise` but I'm not sure.

Or maybe it doesn't matter. 🙂

Reply:
I guess triggering is the way to go. Would that sound better?

```
For example, imagine a stateless `Button` component that triggers an event when clicked.
```